hw_16_todo/delete.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+require_once __DIR__ . '/db.php';
+
+session_start();
+
+if (isset($_POST['id'])){
+    $conn = getConnection();
+
+    $userIdDel = mysqli_real_escape_string($conn, $_POST['id']);
+
+    $sql_del = "Delete From tasks where id = '$userIdDel'";
```

Comment:
лучше еще добавить user_id, чтобы никто кроме автора записи не мог удалить
```suggestion
    $sql_del = "Delete From tasks where id = '$userIdDel' and user_id= '....'";
```